assets/js/components/SettingsNotice.js

```diff
@@ -22,30 +22,112 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+/**
+ * Internal dependencies
+ */
+import InfoIcon from '../../svg/info-icon.svg';
+import SuggestionIcon from '../../svg/suggestion-icon.svg';
+import WarningIcon from '../../svg/warning-icon.svg';
+
 export const TYPE_WARNING = 'warning';
 export const TYPE_INFO = 'info';
 export const TYPE_SUGGESTION = 'suggestion';
 
-export default function SettingsNotice( { children, type } ) {
+const getIconFromType = ( type ) => {
+	switch ( type ) {
+		case 'warning':
+			return <WarningIcon />;
+		case 'info':
+			return <InfoIcon />;
+		case 'suggestion':
+			return <SuggestionIcon />;
+		default:
+			return null;
+	}
+};
+
+const SettingsNoticeSingleRow = ( {
```

Comment:
Normally I would put new components in their own file but I'm not sure what to do here when all the components are in their own folder :thinking:

Reply:
Yes, please, put all components into separate files. You can create a new folder `assets/js/components/SettingsNotice/`.